auth-web/src/store/modules/business.ts

```diff
@@ -218,20 +218,45 @@ export default class BusinessModule extends VuexModule {
 
   // Following searchBusiness will search data from legal-api.
   @Action({ rawError: true })
-  public async searchBusiness (businessIdentifier: string): Promise<any> {
-    const response = await BusinessService.searchBusiness(businessIdentifier)
-    if (response && response.data && response.status === 200) {
+  public async searchBusiness (businessIdentifier: string): Promise<LearBusiness> {
+    const response = await BusinessService.searchBusiness(businessIdentifier).catch(() => null)
+    if (response && response?.status === 200 && response?.data?.business?.legalName) {
       ConfigHelper.addToSession(SessionStorageKeys.BusinessIdentifierKey, businessIdentifier)
       return response.data.business
+    } else {
+      throw Error('search failed')
     }
-    return null
   }
 
   @Action({ rawError: true })
   public async createBNRequest (request: BNRequest): Promise<any> {
     return BusinessService.createBNRequest(request)
   }
 
+  @Action({ rawError: true })
+  public async getBNRequests (businessIdentifier: string): Promise<RequestTracker[]> {
+    const response = await BusinessService.getBNRequests(businessIdentifier).catch(() => null)
+    if (response && response?.status === 200) {
+      return response.data.requestTrackers
+    }
+    return []
+  }
+
+  @Action({ rawError: true })
+  public async resubmitBNRequest (resubmitRequest: ResubmitBNRequest): Promise<any> {
+    const response = await BusinessService.resubmitBNRequest(resubmitRequest).catch(() => null)
+    return response && (response?.status === 200 || response?.status === 201)
```

Comment:
This can be simplified with conditional chaining.

Does the caller handle the different return types (could be True, False or Null) ?